src/stylesheets/objects/_content-layouts.scss

```diff
@@ -66,6 +66,10 @@
   @include block;
 }
 
+[class*="block accent"] {
```

Comment:
This will only work if the classes are listed as `block accent`. Try something like:
`[class*="block"][class*="accent"]` for anything with those two classes - such as `class="block big accent"`.